lib/src/components/NxPagination/__tests__/NxPagination.test.tsx

```diff
@@ -302,16 +334,14 @@ describe('NxPagination', function() {
     testWithProps({ pageCount: 6, currentPage: 5 });
   });
 
-  it('adds tabIndex=-1 to the selected page and either tabIndex=0 or no tabIndex to all other buttons', function() {
+  it('adds either tabIndex=0 or no tabIndex to all other buttons', function() {
```

Comment:
More accurate to what you have at the moment: `adds tabIndex=0 to all buttons`.  However as I mentioned above I'm not sure you even need to do that.  If you don't, this test can just be removed.

Reply:
I removed this test entirely in e9672ddf911cb578dece43342b812ef76719e189. I don't see any reason to keep it since we don't need to have the tabIndex set on any of the buttons.